src/frontend/src/pages/FinancePage/ReimbursementRequestsSection.tsx

```diff
@@ -41,12 +115,85 @@ const ReimbursementRequestTable = ({
           <TableHead>
             <TableRow>
               {tabValue === 1 && <ColumnHeader title="Recipient" />}
-              <ColumnHeader title="Sabo ID" />
-              <ColumnHeader title="Amount ($)" />
-              <ColumnHeader title="Date Submitted" />
-              <ColumnHeader title="Date Submitted To Sabo" />
-              <ColumnHeader title="Vendor" />
-              <ColumnHeader title="Status" />
+              <TableCell
```

Comment:
Let's make a separate component definition (or helper function) for these similar `TableCell` components, so we can shorten the code in this file. 

We can make a new .tsx file in FinancePage/FinanaceComponents and create a separate component in it. We can refer to ColumnHeader.tsx as an example in making separate small component.